### DISCUSSION TITLE
if you are in search of star 👇

### DISCUSSION THREAD

**Discussion by csthenry** [Schweinepriester/github-profile-achievements#246] [Q&A]
please leave a star here 🙏🙏 : https://github.com/csthenry/MagicLitePlanet
i will help others in search of star,
btw thanks in advance

~ add a star and add leave your repo link here.

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by csthenry** [reply]
Please leave a star here https://github.com/csthenry/MagicLitePlanet before.

**Reply by cesarreyes3** [reply]
@csthenry he never left a star for anyone, but he always asks for that

**Reply by csthenry** [reply]
@cesarreyes3 so, I didn't leave a star for him

**Reply by erikn69** [reply]
+1
https://github.com/Schweinepriester/github-profile-achievements/discussions/208#discussioncomment-3208883

**Reply by Panchito6w62** [reply]
No tengo ando frío

**Reply by ArielMAJ** [reply]
@Panchito6w62I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by cesarreyes3** [comment]
Done, please let me one here
https://github.com/masbug/flysystem-google-drive-ext

**Reply by csthenry** [reply]
done.

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by ArielMAJ** [reply]
@cesarreyes3 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by erikn69** [comment]
Done, 
Please help here: https://github.com/erikn69/laravel-google-drive-ext-demo

**Reply by csthenry** [reply]
thank u,
done.

**Reply by iamirhoseinr** [reply]
:heart:

**Comment by Chameleon-Coder** [comment]
Chameleon-Coder/Chameleon-Coder

**Reply by csthenry** [reply]
ok

**Comment by csthenry** [comment]
anticipate your participation!

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by sulincix** [comment]
Please help here: https://github.com/sulincix/debian-subsystem

debian subsystem for linux :D

**Reply by csthenry** [reply]
Please leave a star here https://github.com/csthenry/MagicLitePlanet before.

**Reply by sulincix** [reply]
done

**Reply by csthenry** [reply]
thank u，i will.

**Reply by cesarreyes3** [reply]
@csthenry he did remove his star from you

**Reply by csthenry** [reply]
Thanks, I also removed the star for him, he's untrustworthy.

**Comment by lokesh7suman** [comment]
https://github.com/d4rkvent/fonts.git

**Comment by zigad** [comment]
Just created a Playground Repo for anyone wishing to play and try new stuff on GitHub without any fear of breaking anything.

Feel free to push, merge, comment, open issues and so on:

https://github.com/zigad/Playground

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Comment by EeeasyCode** [comment]
https://github.com/EeeasyCode/gachon-discord-crawling-bot
plz come to my repo
Im learning to programing
Im study hard to south korea
Thanks!!

**Reply by csthenry** [reply]
thank you.

**Reply by PaulSpoerry** [reply]
Starred... trade ya! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by bensonarafat** [reply]
done! https://github.com/codepymdev/grad

**Comment by PaulSpoerry** [comment]
Starswap! https://github.com/PaulSpoerry/test

**Reply by csthenry** [reply]
ok

**Comment by bacca87** [comment]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by PaulSpoerry** [reply]
Done... trade please: https://github.com/PaulSpoerry/test

**Reply by bensonarafat** [reply]
Done, for me too https://github.com/codepymdev/grad

**Comment by sbsoso0411** [comment]
@csthenry 
done, please start my repo. https://github.com/fullstack-webdev/Noexam-WordPress thank you

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by bensonarafat** [reply]
pls do mine

**Comment by 3kh0** [comment]
I need a answer, mark this as answer please, I gave you a star

**Reply by sbsoso0411** [reply]
done

**Reply by 3kh0** [reply]
@fullstack-webdev can you star some of my repos thanks so much!

**Reply by sbsoso0411** [reply]
:) Done. I did give a star at 2 repos.

**Reply by 3kh0** [reply]
thanks @fullstack-webdev!

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by ArielMAJ** [comment]
@csthenry I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@csthenry I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again!

**Comment by bm4578** [comment]
Please leave a star here:
https://github.com/bm4578/docker.git

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad  
done pls do for me

**Comment by PARKER-X** [comment]
https://github.com/PARKER-X/BarBar

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad 
thanks

**Comment by PARKER-X** [comment]
carpido/home at main · PARKER-X/carpido

**Comment by bensonarafat** [comment]
https://github.com/codepymdev/grad

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Comment by om3ga6400** [comment]
@csthenry https://github.com/OMEGA6405/OMEGA6405

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)
